### DISCUSSION TITLE
how to add buffur in audio start and end in 7.0.0beta.6

### DISCUSSION THREAD

**Discussion by umairmanzoor1143** [katspaugh/wavesurfer.js#2864] [General]
And what about add buffur in audio start and end in 7.0.0beta.6
https://codesandbox.io/s/sharp-johnson-4po7pn?file=/src/useWavesurfer.tsx
can you please check this.
like this :
https://codepen.io/selfiepop/pen/wvQvKNx?editors=1111
this is on 6.6.3 version

**Comment by katspaugh** [comment]
In v7, `wavesurfer.getDecodedData()` will give you an audio buffer. You can then manipulate it like in your v6 code, and and load it again like this:
`wavesurfer.load(URL.createObjectURL(new Blob([buffer], 'audio/wav'))`

**Reply by umairmanzoor1143** [reply]
am exactly doing that
